ci/vars-functional-test.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
```suggestion
---
# temp: skip os-net-setup
cifmw_os_net_setup_config: []
```

The os_net_setup tasks are conflicting with the `make edpm_instance_deploy` task.
We will have to properly fix this so that the resources don't conflict.